common/Stress/Azure.Test.Stress/StressProgram.cs

```diff
@@ -44,6 +44,26 @@ public static async Task Main(Assembly assembly, string[] args)
 
         private static async Task Run(Type testType, StressOptions options)
         {
+            // Test types must contain a public constructor SampleTest(SampleOptions, SampleMetrics)
+            Type metricsType = null;
+            foreach (var ctor in testType.GetConstructors())
```

Comment:
This is also very similar to the snippet in `PerfProgram` and `PerfStressUtilities`....  I wonder if its possible to abstract things out and share code.